src/encoded/static/components/filegallery.js

```diff
@@ -2163,7 +2163,11 @@ export const compileAnalyses = (experiment, files) => {
             });
         }
     }
-    return _(compiledAnalyses).sortBy(compiledAnalysis => -compiledAnalysis.assemblyAnnotationValue);
+
+    // In ENCODE releases, release year is not in the data. To get around this, if you strip out the text and
+    // leave only numbers, you will consistently have a means to identify ealier releases from latter ones. This
+    // approached was used to sort. "Lab custom" appear at the bottom as intented
+    return _(compiledAnalyses).sortBy(a => parseInt(a.title.replace(/\D/g, ''), 10)).reverse();
```

Comment:
I feel like it’s sort of a shame that we do the sorting in `computeAssemblyAnnotationValue` and then ignore it with this sort. Could you instead do a two-key sort, with `computeAssemblyAnnotationValue` doing the first stage, and then sort those by project? What do you think?

Reply:
@forresttanaka 
Do you mean:

```
 return _(compiledAnalyses)
        .sortBy(compiledAnalysis => -compiledAnalysis.assemblyAnnotationValue)
        .sortBy(a => parseInt(a.title.replace(/\D/g, ''), 10)).reverse();

```
That is doable but I believe the second sort will override the first

Reply:
Yeah, this still overrides the first sort. If you could just sort on the project/rfa without affecting the order within each project/rfa that compiledAnalysis did, I think that would be great. I think `computeAssemblyAnnotationValue` could result in something like:

ENCODE3 GRCh38
ENCODE4 GRCh38
ENCODE3 hg19
ENCODE4 hg19

And then if you could keep that existing sorting while sorting on the project/rfa:

ENCODE4 GRCh38
ENCODE4 hg19
ENCODE3 GRCh38
ENCODE3 hg19

Right now your `replace` redoes the sorting that `computeAssemblyAnnotationValue`, making calling `computeAssemblyAnnotationValue` pointless. I used `computeAssemblyAnnotationValue` because the assembly/annotation facet in the file gallery uses this same order. So if that function needs modification for some future assembly/annotation, then this menu follows that order within a project/rfa.

Reply:
Is this along the lines you were thinking?